docroot/modules/custom/json_data_field/src/Plugin/Field/FieldFormatter/JsonDataFieldFormatter.php

```diff
@@ -29,6 +29,7 @@ class JsonDataFieldFormatter extends FormatterBase implements ContainerFactoryPl
    */
```

Comment:
Just noting the last file I made in the review so far in case Github forgets what I have seen so far.